client/dead.lua

```diff
@@ -125,6 +125,7 @@ AddEventHandler('gameEventTriggered', function(event, data)
         DeathTime = config.deathTime
         OnDeath()
     end
+    TriggerEvent('qbx_medical:client:attackerData', attacker, weapon)
```

Comment:
Pass this data to the OnDeath function and to the existing onPlayerDied event. No need for a new event. https://github.com/Qbox-project/qbx_medical/blob/380465254d4af3f1c8c6d6e4c2f7ea0c3645647f/client/dead.lua#L26